### ISSUE TITLE
Tree of life for MetaPhlAn3 species

### ISSUE THREAD

**Issue by alexhbnr** [biobakery/phylophlan#25]
Hi,

I regularly run bacterial composition analysis using UniFrac distances based on the relative abundances returned by MetaPhlAn. The UniFrac algorithm requires a phylogenetic tree describing the distance between the phylogenetic analysis and so far I always used the tree as part of the Bioconductor package `CuratedMetagenomicData` that is returned by its function `getMetaphlanTree`. According to its help, it tree was created using PhyloPhlAn by you, @fasnicar .

Due to the update to the new ChocoPhlAn database, there are now many more species in the database of MetaPhlAn and the tree present in `CuratedMetagenomicData` does not included all species. Therefore, I extracted all genomes from the MetaPhlAn pickle file (of database version v293), selected one representative per species, and ran PhyloPhlAn3 with the option `--diversity high` and `--accurate` using the PhyloPhlAn marker database. After trimming and everything, I concatenate all alignments into one super-alignment and ran IQtree to get the [tree](https://shh-cloud.gnz.mpg.de/index.php/s/SyAWfiytfnAdZm6)

PhyloPhlAn3 returns a tree including most of the input species. A few of them were discarded because of the low information content of the alignment of these species. Most of the closely related species form species-specific clades but the there is definitely conformity that all archaea form the a single monophyletic clade etc.

My question: did you do something differently to have such a clean tree of life based on the species present in MetaPhlAn?

**Comment by fasnicar** [comment]
Hello Alex, 
For the phylogenetic tree released in curatedMetagenomicData we did use PhyloPhlAn, and what we did was to first select the highest quality representative genome for each species, we used to slowest and most accurate set of parameters for MAFFT, and instead of using IQ-TREE we used FastTree to build a backbone phylogeny and then RAxML for refining the placement.

We are going to re-build a new tree for the new MetaPhlAn 3.0, but I don't have an eta about when this will be done.

Many thanks,
Francesco

**Comment by alexhbnr** [comment]
Hi Francesco,

Thanks for the feedback. When selecting the representative genomes, I also evaluated at which assembly level the individual genomes for each species had and selected the one of highest quality, so I guess that should be less of an issue.

I will try your suggestions regarding alignment and using a two-step strategy to gain the accurate phylogeny. Initially, I used FastTree and RAxML on a per marker approach but then Astral failed merging the per-marker trees into a consensus tree. Using a backbone tree might solve this.

Thanks for your suggestions!
